engine/util/util_test.go

```diff
@@ -164,3 +168,77 @@ func TestStructTagToFieldName2(t *testing.T) {
 	t.Logf("got output: %+v", m)
 	t.Logf("got error: %+v", err)
 }
+
+func TestStructKindToFieldNameTypeMap(t *testing.T) {
```

Comment:
Can you move this test to the bottom of the file after the declaration of the resource please? Or even better, put it _inside_ the test block like how the other tests do?

Reply:
> put it inside the test block like how the other tests do? 

Unless I understand your meaning, struct functions can't be declared inside functions. If you have tests you can link for reference to explain the requested change, that would help

Reply:
The test was relocated beneath the resource in 8ef2386

Reply:
> Unless I understand your meaning, struct functions can't be declared inside functions. If you have tests you can link for reference to explain the requested change, that would help

You are entirely correct, my mistake sorry. Standalone structs can, but I didn't realize the methods on them can't be! Woops!